src/sparseml/pytorch/utils/quantization/quantize_qat_export.py

```diff
@@ -508,21 +516,219 @@ def _convert_quantizable_gemm(
         remove_node_and_params_from_graph(model, gemm_node, keep_params=params_to_keep)
 
 
+def _convert_quantizable_matmul_and_add(model: ModelProto):
+    """
+    A pass for converting a MatMul with kernel and bias into a quantized representation
+
+    Starting with:
```

Comment:
This isn't going to convert well for docs, need to preface each line with | to make sure the formatting is kept exactly as shown in here